crates/bitwarden/src/uniffi_support.rs

```diff
@@ -0,0 +1,62 @@
+use std::{num::NonZeroU32, str::FromStr};
+
+use uuid::Uuid;
+
+use crate::{crypto::CipherString, error::Error, UniffiCustomTypeConverter};
+
+uniffi::custom_type!(NonZeroU32, u32);
+
+impl UniffiCustomTypeConverter for NonZeroU32 {
```

Comment:
@dani-garcia I re-wrote these to use the new macro from uniffi.

Reply:
Nice, looks much simpler now with the latest uniffi changes!